tt_eager/tt_dnn/op_library/operation.hpp

```diff
@@ -218,7 +218,7 @@ struct HostOperation final {
 };
 
 struct DeviceOperation final {
-    using storage_t = std::array<std::byte, 768>;
+    using storage_t = std::array<std::byte, 816>;
```

Comment:
what is this mysterious number? can you add a comment about it?

Reply:
I think it's equivalent to the size in bytes of Memory_Config. I am not sure however, @arakhmati is that the case?

Reply:
DeviceOperation uses type erasure to store the underlying operation struct. And the storage must be as big as the biggest underlying operation struct.